puppet/modules/web/manifests/vhost/web.pp

```diff
@@ -26,12 +26,14 @@
     { 'rewrite_rule' => ['^/wiki(.*)  https://projects.theforeman.org/wiki$1  [R,L]'] },
     { 'rewrite_rule' => ['^/events/all.ics https://community.theforeman.org/c/events/l/calendar.ics [R,L]'] },
     { 'rewrite_rule' => ['^/events/? https://community.theforeman.org/c/events/l/calendar [R,L]'] },
+    { 'rewrite_rule' => ["^/api/latest(.*) https://apidocs.theforeman.org/foreman/${stable}\$1 [R,L]"] },
+    { 'rewrite_rule' => ['^/api/(.*) https://apidocs.theforeman.org/foreman/\$1 [R,L]'] },
```

Comment:
Doesn't look like this works:
```console
$ curl https://theforeman.org/api/2.4/apidoc/v2/compute_profiles/create.html -si | grep Location
Location: https://apidocs.theforeman.org/foreman/$1
```

Should have been just `$1`.

Reply:
https://github.com/theforeman/foreman-infra/pull/1608

Reply:
Damn sorry!